packages/schematics/angular/application/files/standalone-files/src/app/app.config.ts.template

```diff
@@ -0,0 +1,9 @@
+import { ApplicationConfig } from '@angular/platform-browser';<% if (routing) { %>
```

Comment:
The `ApplicationConfig` symbol was moved to the `@angular/core` recently: https://next.angular.io/api/core/ApplicationConfig.
```suggestion
import { ApplicationConfig } from '@angular/core';<% if (routing) { %>
```